pandas/plotting/_core.py

```diff
@@ -3500,6 +3615,12 @@ def scatter(self, x, y, s=None, c=None, **kwds):
             - A column name or position whose values will be used to color the
               marker points according to a colormap.
 
+        size_factor : scalar, optional
+            A multiplication factor to change the size of bubbles
```

Comment:
Does this only apply when `s` is a column name? If so, state that.

Reply:
It applies to all cases, not only when `s` is a column name. Which is also useful for simple scatter plots with constant marker size by the way, as it is probably more intuitive to write `size_factor=2` than `s=40` if you want to double the size of markers.